docs/docs/tutorial.md

```diff
@@ -488,17 +478,21 @@ Hence, we will be using `this.state` to save the user's input as it is entered.
 
 ```javascript{3-11,15-26}
 // tutorial16.js
-var CommentForm = React.createClass({
-  getInitialState: function() {
-    return {author: '', text: ''};
-  },
-  handleAuthorChange: function(e) {
+export class CommentBox extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      author: '', 
+      text: ''
+    }
+  }
+  handleAuthorChange = (e) => {
```

Comment:
Let's put a pin in this for the time being to prevent over-churn. I don't know that we will be encouraging the use of property initializers yet. We'll want to make sure we have some guidelines for code in documentation before we finalize this.